src/workers/jobs.js

```diff
@@ -267,6 +271,7 @@ export async function uploadContacts(job) {
     .then(result => {
```

Comment:
I wonder if the next step is to add a `.catch(err => { console.log('DEBUG: uploadContacts error', campaignId, err)})`
after the .then() -- maybe the error is happening there and we're not getting it -- alternatively or in addtion to, we could try...catch the top await call to get the count.

Reply:
Okay, added some error catching. Hopefully it make sense!